File: Code/js/pattern9.js
limit = prompt("Enter Limit");

if (limit % 2 == 1)
{
  for (i = 0; i * 2 + 1 <= limit; i++)
    console.log("   ".repeat(Math.floor(limit / 2) - i) + " * ".repeat(i * 2 + 1));
  for (i = Math.floor(limit / 2); i > 0; i--)
    console.log("   ".repeat(Math.ceil(limit / 2) - i) + " * ".repeat(i * 2 - 1));
}
else
  console.log("Enter Odd Number");

File: Code/py/pattern9.py
limit = int(input("Enter Limit: "))

if limit % 2 == 1:
    for i in range(1, limit // 2 + 1):
        print("   " * (limit // 2 - i + 1) + " * " * (2 * i - 1))
    for i in range(limit // 2 + 1, 0, -1):
        print("   " * (limit // 2 - i + 1) + " * " * (2 * i - 1))
else:
    print("Enter Odd Number")